### PR TITLE
Post data used for entering data in the database collections.

### PR DESCRIPTION
**Pull request by hemant6488**
post requests are used to add data to a collection in mongoDB, and then print a response that the addition of data was successful

### CODE REVIEW

**Comment by kartikjain** [comment]
hey plz remove the postman_example.png file

**Comment by psych0der** [comment]
Try to make commits with meaningful messages.

On Saturday, January 3, 2015, Kartik Jain <notifications@github.com> wrote:

> hey plz remove the postman_example.png file
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mashproject/NGOConnect-Api-server/pull/2#issuecomment-68590067>
> .
>


-- 
Regards
*Mayank Bhola*

File: app.js [hunk 1]
```diff
@@ -21,6 +21,23 @@ app.all('*', function(req, res, next) {
...
+//Handling post requests for this URL, calling a function from stickies.js.
+
+app.post('/pinboard', stickies.saveEntry);
+
+/*
+changes end
+*/
+
+
+
```
Comment (kartikjain) [inline app.js]: These extra spaces need to be removed

File: create_database.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+mdb.ngo.save(ng3, function(err, savedngo){
+	if(err) console.log(err);
+	else console.log("ngo "+ savedngo.ngoname +" saved")
+});
+
+mdb.ngo.save(ng4, function(err, savedngo){
+	if(err) console.log(err);
+	else console.log("ngo "+ savedngo.ngoname +" saved")
+});
```
Comment (kartikjain) [inline create_database.js]: Always add 1 blank line at the end of each file or you will see this red error message

Comment (kartikjain) [inline create_database.js]: Also this file will go in our wiki rather than in main code

File: README.md [hunk 3]
```diff
@@ -1,4 +1,72 @@
...
+
+------------------------------------------------------------------------------------------------------------------------------------------------
+
+####Integration with NodeJS Server:####
+
+1. MongoJS is a very simple wrapper for MongoDB driver for NODEJS, it mimics the actual MongoDB Api very closely.
+2. Grab MongoJS : npm install mongojs
+3. Require MongoJS and assign it to an object, var db = require('mongojs');.
+4. Now, db.connect(dburl, collections); //here dburl = "localhost/databasename" //Collections: similar to tables RDB, used to group similar data together.
+
```
Comment (kartikjain) [inline README.md]: Till this point ..the steps should be added to the create_database.js file in our wiki.
After this part the content is not needed.create_database is sufficiet.

**Pull request closed without merging**